### PR TITLE
CDSR-805 - refactor model for movementreferencenumber

### PR DESCRIPTION
**Pull request by caoilte**
- Removed Answers for enter movement reference
number page
- Corresponding Backend PR: https://github.com/hmrc/cds-reimbursement-claim-frontend/pull/289

### CODE REVIEW

**Comment by mikail-khan** [comment]
Were you able to launch the service and submit a claim as a test?

**Comment by caoilte** [comment]
> Were you able to launch the service and submit a claim as a test?

yes. I've tested new backend and new frontend together locally and could submit a claim. UI tests fail when run locally but I think that's because they're currently broken in master. Regardless I'll hold off on merging once approved until I can see them working.

**Review by aneksamun — APPROVED** [review]

**Pull request merged**
